content/200-concepts/100-components/01-prisma-schema/06-relations/500-troubleshooting-relations.mdx

```diff
@@ -259,3 +259,13 @@ model Category {
   posts Post[] @relation("CategoryPostRelation")
 }
 ```
+
+## Using m-n relations in databases with enforced primary keys
+
+### Problem
+
+Some cloud providers enforce (by default) the existence of primary keys in all tables. However, any Join tables created by Prisma for m-n relations using implicit syntax in Prisma Schema do not have primary keys. 
+
+### Solution
+
+If your database provider automatically creates a primary key, and you want to use m-n relations, you need to use explicit relation syntax, manually create the join model, and verify that this join model has a primary key. 
```

Comment:
Same comment as in the other PR. https://github.com/prisma/docs/pull/2469#pullrequestreview-797898457

```suggestion
You need to use explicit relation syntax, manually create the join model, and verify that this join model has a primary key. 
```

Reply:
I think we should consider adding a link to the explicit relation syntax, here, perhaps? Unless it's really easy to navigate to (scroll up or down to a section that is close).